### PR TITLE
[calceph] update to version 3.5.1

### PR DESCRIPTION
**Pull request by gastineau**
**Describe the pull request**

- #### What does your PR fix?  
Update the version to 3.5.1

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
all, No

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  `Your answer`

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
 Yes
and I run before "./vcpkg format-manifest ports/calceph/vcpkg.json"

**If you are still working on the PR, open it as a Draft: https://github.blog/2019-02-14-introducing-draft-pull-requests/**

### CODE REVIEW

File: ports/calceph/vcpkg.json [hunk 1]
```diff
@@ -1,9 +1,10 @@
 {
   "name": "calceph",
-  "version": "3.5.0",
-  "port-version": 2,
+  "version": "3.5.1",
+  "port-version": 1,
```
Comment (Thomas1664) [inline ports/calceph/vcpkg.json:4]: `port-version` can be removed if `version` has changed. After committing the change use `vcpkg x-add-version --overwrite-version` to update the version database.

File: versions/c-/calceph.json [hunk 2]
```diff
@@ -1,5 +1,15 @@
 {
   "versions": [
+    {
+      "git-tree": "2afb6f77bd690fad5c6adfd7a75c4241c7cb1c13",
+      "version": "3.5.1",
+      "port-version": 1
+    },
```
Comment (Thomas1664) [inline versions/c-/calceph.json:7]: Please remove this.

**Review by Thomas1664 — CHANGES_REQUESTED** [review]

File: versions/c-/calceph.json (hunk 2 above)
Comment (gastineau) [reply versions/c-/calceph.json]: done

**Review by Thomas1664 — APPROVED** [review]

**Comment by gastineau** [comment]
pipelines fail on this error: " msiexec failed while extracting 'D:\downloads\7z1900-x64.msi'". It seems to be related to #23284

**Review by LilyWangLL — APPROVED** [review]

**Pull request merged**